data/successors/successor 1 introduction.txt

```diff
@@ -0,0 +1,1103 @@
+			`	"'Tis no surprise," Sieasej says, "that 'twere such valued pets if their demeanor is so docile and agreeable." They tuck their triangulation device away in a pocket and slither with you as you approach your ship.`
+				goto ambush
+			label tranq-friends
+			`	While the animal is distracted, you take aim and fire a shot from your <tranq h2h> at it. It briefly jumps to escape but soon plummets to the ground in a puff of leaves and dusty feathers.`
+				goto bring-tranq
+			label net-friends-alarmed
+			`	While the animal is distracted, you toss a net over it and it falls to the ground, where it shrieks for a duration before going quiet. The shriek is soon echoed by similar ones from other animals in the area, and once those go quiet as well, your ears are pierced by a loud, warbling wail in the distance.`
+				goto escape
+			label bring-tranq
+			`	You scoop up the tranquilized animal, careful not to damage its extremities, and begin the walk back to your ship.`
+				goto ambush
+			label escape
+			choice
+				`	(Grab it and run!)`
+			`	You scoop up the immobilized animal and begin to run back to your ship as the wailing behind you increases in pitch and intensity. As you and Sieasej dash through the foliage, the sunlight overhead is suddenly darkened by the shadow of something flying overhead.`
+			choice
+				`	(Keep running.)`
+				`	(Try to shoot at whatever is above you.)`
+					goto shoot-ambush
+			`	Suddenly, the shadow disappears and the wailing stops. You keep running towards your ship, but can't shake the feeling that you are being watched.`
+				goto ambush
+			label shoot-ambush
+			`	You draw your sidearm and look to see what's above you, but the shadow is suddenly gone and you are left staring up into the sunlight and leaves. The wailing has stopped as well; indeed, the whole area remains eerily quiet as you continue to run towards your ship.`
+			label ambush
+			`	When you arrive, however, you find that blocking your entry to the <ship> are two large creatures. Their glassy skin resembles that of the smaller creature you captured, but they are each several meters in size. Their feelers, rather than soft and downy, are whiplike and serrated and tipped with vicious-looking barbs that they thrust towards you and Sieasej upon noticing your presence. Their many eyes flick back and forth between you and the animal that you captured, and the two let out a loud wail in unison. Waves of red and blue run down Sieasej's body, probably an expression of fear.`
+			choice
+				`	(Run for it!)`
+				`	(Shoot them with your sidearm.)`
+					goto shoot
+				`	(Order your crew to fire a turret at them.)`
+					to display
+						"flagship base attribute: turret mounts" > "flagship attribute: turret mounts"
```

Comment:
Not every turret is hostile; AM might not be useful. Also, turrets like a Firestorm Battery would obliterate everything nearby, maybe even your ship.

Reply:
I got the idea for this action from the Bunrodea first contact mission, where the same condition is used to check if First Last can disable a Bunrodea anti-repulsor vehicle on the ground. It's a bit of an abstraction in both cases, but I like the additional option. Perhaps I should be more vague about the weapon's effects?